docs/eventing/sink/kafka-sink.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Apache Kafka Sink"
+weight: 30
+type: "docs"
+---
+
+## Prerequisites
+
+[Knative Eventing installation](./../../install/any-kubernetes-cluster.md#installing-the-eventing-component).
+
+## Installation
+
+1. Install the Kafka controller by entering the following command:
+
+    ```bash
+    kubectl apply --filename {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-controller.yaml" >}}
+    ```
+
+1. Install the Kafka Sink data plane by entering the following command:
+
+    ```bash
+    kubectl apply --filename {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-sink.yaml" >}}
+    ```
+
+1. Verify that `kafka-controller` and `kafka-sink-receiver` are running,
+by entering the following command:
+
+    ```bash
+    kubectl get deployments.apps -n knative-eventing
+    ```
+
+    Example output:
+
+    ```bash
+    NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
+    eventing-controller            1/1     1            1           10s
+    eventing-webhook               1/1     1            1           9s
+    kafka-controller               1/1     1            1           3s
+    kafka-sink-receiver            1/1     1            1           5s
+    ```
+   
+## Create a Kafka Sink
+
+A Kafka Sink object looks like this:
```

Comment:
I would move this conceptual info to the sink readme file and just have the heading "Kafka sink" for a section there.
"## Create a Kafka Sink" would lead me to expect steps under this heading rather than conceptual info first.